template.tpl

```diff
@@ -0,0 +1,1704 @@
+          },
+          {
+            "type": "TEXT",
+            "name": "commonEventNest",
+            "displayName": "Nest common event properties under key (optional)",
+            "simpleValueType": true,
+            "enablingConditions": [
+              {
+                "paramName": "includeCommonEventProperties",
+                "paramValue": true,
+                "type": "EQUALS"
+              }
+            ],
+            "help": "Specify a key under which the Common Event properties will be nested. Alternatively, leaving it blank adds the the common event properties in the request body without nesting."
+          },
+          {
+            "type": "CHECKBOX",
+            "name": "includeCommonUserProperties",
+            "checkboxText": "Include common user properties",
+            "simpleValueType": true,
+            "defaultValue": false,
+            "alwaysInSummary": false,
+            "help": "Include the `user_data` properties from the common event definition in the request body."
+          },
+          {
+            "type": "TEXT",
+            "name": "userDataNest",
+            "displayName": "Nest user_data under key (optional)",
+            "simpleValueType": true,
+            "enablingConditions": [
+              {
+                "paramName": "includeCommonUserProperties",
+                "paramValue": true,
+                "type": "EQUALS"
+              }
+            ],
+            "help": "Specify a key under which the `user_data` properties from the common event will be nested. Alternatively, leaving it blank adds the `user_data` in the request body without nesting."
+          },
+          {
+            "type": "SIMPLE_TABLE",
+            "name": "eventMappingRules",
+            "displayName": "Additional Event Property Mapping Rules",
+            "simpleTableColumns": [
+              {
+                "defaultValue": "",
+                "displayName": "Event Property Key",
+                "name": "key",
+                "type": "TEXT",
+                "valueValidators": [
+                  {
+                    "type": "NON_EMPTY"
+                  }
+                ]
+              },
+              {
+                "defaultValue": "",
+                "displayName": "Destination Mapped Key (optional)",
+                "name": "mappedKey",
+                "type": "TEXT"
+              }
+            ],
+            "alwaysInSummary": false,
+            "help": "Specify the Property Key from the GTM Event, and then key you could like to map it to or leave the mapped key blank to keep the same mapping. You can use Key Path notation here (e.g. `x-sp-tp2.p` for a Snowplow events platform or `x-sp-contexts.com_snowplowanalytics_snowplow_web_page_1.0.id` for a Snowplow events page view id (in array index 0)."
```

Comment:
It might be worth updating this help text to include a not that you can also use dot notation in the `Destination Mapped Key (optional)` values too.

Reply:
That would be useful, indeed! Added a note to this and also to the help texts of all other places where the dot notation can be used.